api/tests/Behat/bootstrap/v2/Common/BaseFeatureContext.php

```diff
@@ -29,7 +31,7 @@ class BaseFeatureContext extends MinkContext
 
     public const BEHAT_FRONT_RESET_FIXTURES = '/behat/frontend/reset-fixtures?testRunId=%s';
     public const BEHAT_FRONT_USER_DETAILS = '/behat/frontend/user/%s/details';
-    public const REPORT_SECTION_ENDPOINT = '%s/%s/%s';
+    public const REPORT_SECTION_ENDPOINT = '/%s/%s/%s';
```

Comment:
Was a bit weary about this looks ok as handled by mink which can do either. 👍

Reply:
Yeah I had a failing test for some reason due to the missing first slash which is odd as it must have passed for you?